### PR TITLE
RFC: Reduce Build Complexity of .Net Runtimes

### PR DESCRIPTION
**Pull request by ForestEckhardt**
[Readable](https://github.com/paketo-buildpacks/rfcs/blob/simplify-dotnet-runtime/text/dotnet-core/0005-simplify-runtime-dependency.md)

### CODE REVIEW

File: text/dotnet-core/rfcs/0005-simplify-runtime-dependency.md
Comment (sophiewigmore) [inline text/dotnet-core/rfcs/0005-simplify-runtime-dependency.md:11]:
```suggestion
launch using our custom dependencies, we need to build a somewhat convoluted
```

**Comment by ForestEckhardt** [comment]
@fg-j I think I would like to do this as a smaller chuck of work that would make the re-ordering easier to do in the future.

**Comment by sophiewigmore** [comment]
IMO it would be nice to combine all of these concerns into one RFC

**Comment by ForestEckhardt** [comment]
I have spent a little time looking at the re-architecture and I feel like I would just need to tack an additional RFC on to the end of this RFC that is contingent on this RFC passing. I am happy to do that but I think that the purposed re-architecture that @fg-j is purposing becomes a trivial RFC that can reference this RFC and the SDK RFC once they both pass. Please let me know you would like me know what y'all are feeling and I will go with the popular vote.

**Comment by fg-j** [comment]
+1 I'd like it all to happen in this RFC. Gives us an opportunity to evaluate if there are edge cases we're missing and how we could cover them.

**Review by sophiewigmore — APPROVED** [review]

**Comment by joshuatcasey** [comment]
2022-07-26 Working Group notes: waiting for approvals

File: text/dotnet-core/0005-simplify-runtime-dependency.md
Comment (ForestEckhardt) [inline text/dotnet-core/0005-simplify-runtime-dependency.md:79]:
```suggestion
(`dotnet-core-aspnet-runtime`).  This mimics the name of the dependency as it is provided
```

File: text/dotnet-core/0005-simplify-runtime-dependency.md
Comment (fg-j) [reply text/dotnet-core/0005-simplify-runtime-dependency.md:20]:
```suggestion
Runtime hosted by the Paketo dep-server. It constructs a.NET installation and
```

File: text/dotnet-core/0005-simplify-runtime-dependency.md
Comment (fg-j) [inline text/dotnet-core/0005-simplify-runtime-dependency.md:26]:
```suggestion
Core. The .NET Core ASP.NET buildpack adds its dependency to the makeshift .NET
installation.  In other words, for most buildpack builds, the .NET Runtime and .NET
```

File: text/dotnet-core/0005-simplify-runtime-dependency.md
Comment (fg-j) [inline text/dotnet-core/0005-simplify-runtime-dependency.md:34]:
```suggestion
The decision to stitch together a .NET installation was motivated by a desire to build
images with the absolute minimal number of dependencies in them; some .NET apps
```

**Comment by fg-j** [comment]
@paketo-buildpacks/dotnet-core-maintainers Am I good to merge this and get cracking on implementation?

**Comment by dmikusa-pivotal** [comment]
2022-08-02 WG Notes:

Ready to merge.

**Pull request merged**
